Sources/ArcGISToolkit/Components/FeatureFormView/InternalFeatureFormView.swift

```diff
@@ -50,9 +52,11 @@ struct InternalFeatureFormView: View {
                     }
                 }
             }
-            .onChange(of: internalFeatureFormViewModel.featureForm.hasEdits) { hadEdits, hasEdits in
-                if hadEdits && !hasEdits {
-                    internalFeatureFormViewModel.previouslyFocusedElements.removeAll()
+            .task {
+                for await hasEdits in internalFeatureFormViewModel.featureForm.$hasEdits.dropFirst() {
+                    if !hasEdits {
+                        internalFeatureFormViewModel.previouslyFocusedElements.removeAll()
+                    }
```

Comment:
This needed to change as it was relying on re-initialization of `InternalFeatureFormView`. With `NavigationStack` that no longer occurs.